modules/reactor/include/meshgenerators/PolygonMeshGeneratorBase.h

```diff
@@ -401,4 +468,59 @@ class PolygonMeshGeneratorBase : public MeshGenerator
                                              const Real input_origin_x = 0.0,
                                              const Real input_origin_y = 0.0,
                                              const Real tol = 1.0E-10) const;
+
+  /**
+   * Creates bias terms for multiple blocks.
+   * @param radial_biases bias growth factors of the main regions of the blocks
+   * @param intervals radial interval numbers of the main regions of the blocks
+   * @param inner_boundary_layer_fractions radial fractions of the blocks to be defined as the
+   * boundary layers near the inner boundaries
+   * @param inner_boundary_layer_intervals radial interval numbers of the blocks to be defined as
+   * the boundary layers near the inner boundaries
+   * @param inner_boundary_layer_biases bias growth factors of the blocks to be defined as the
+   * boundary layers near the inner boundaries
+   * @param outer_boundary_layer_fractions radial fractions of the blocks to be defined as the
+   * boundary layers near the outer boundaries
+   * @param outer_boundary_layer_intervals radial interval numbers of the blocks to be defined as
+   * the boundary layers near the outer boundaries
+   * @param outer_boundary_layer_biases bias growth factors of the blocks to be defined as the
+   * boundary layers near the outer boundaries
```

Comment:
you say `of the blocks` but I think you mean elements?

Reply:
You are right. Should be `of the elements within the blocks`